fvm/evm/types/codeFinder.go

```diff
@@ -87,3 +88,74 @@ func ExecutionErrorCode(err error) ErrorCode {
 		return ExecutionErrCodeMisc
 	}
 }
+
+func GetErrorForCode(errorCode ErrorCode) error {
```

Comment:
nit: I would call it `ErrorFromCode`, using Get is a bit of anti-pattern https://go.dev/doc/effective_go#Getters

Reply:
Good point :+1:
Updated in https://github.com/onflow/flow-go/pull/5907/commits/db5001a248c5f87061158cda55cc4f222af282b6